src/ResourceManager/Reservations/Commands.Reservations/Cmdlets/GetCatalog.cs

```diff
@@ -17,7 +17,7 @@ public class GetCatalog : AzureReservationsCmdletBase
         public Guid SubscriptionId { get; set; }
 
         [Parameter(Mandatory = true)]
-        [PSArgumentCompleter("VirtualMachines", "SqlDatabases", "SuseLinux")]
+        [PSArgumentCompleter("VirtualMachines", "SqlDatabases", "SuseLinux", "CosmosDb")]
```

Comment:
I don't see updated help documentation with a new valid value. Looks like help may have been generated prior to these changes.

Reply:
Checked in the help document "Get-AzureRmReservationCatalog.md". There are examples. But the document doesn't list all the examples for each reserved resource type. Should have to examples for all the RI types?

Reply:
My fault. I thought that completer values are listed in the reference documentation. You do not need examples for each ReservedResourceType.